### DISCUSSION TITLE
Avatar image in AVIF format

### DISCUSSION THREAD

**Discussion by LITUATUI** [razonyang/hugo-theme-bootstrap#472] [Q&A] [answered]
If can't use an AVIF image for avatar in author.toml.

`avatar = "/images/avatar.avif"`

I get  this error:

`executing "partials/helpers/image.html" at <imageConfig $filename>: error calling imageConfig: image: unknown format`

Am I doing something wrong? Or the theme needs adaptions to support it?

**Comment by razonyang** [comment]
I can't reproduce this issue, could you please provide the avatar you use?

**Reply by LITUATUI** [reply]
Sure, this [one](https://github.com/LITUATUI/m3upt.com/raw/main/static/images/avatar.avif).

I uploaded to a repo so you can download it, but I'm trying in local installation.

**Reply by razonyang** [reply]
@LITUATUI I don't have the permission to access the image(page not found). But I think `avif` images are not supported by Hugo(`imageConfig`). I'll handle it later.

**Comment by razonyang** [comment, marked as answer]
Should be fixed by dd09a8b.
